### PR TITLE
[No JIRA] Upgrade to Node Erbium LTS

### PR DESCRIPTION
**Pull request by k0nserv**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -1 +1 @@
-lts/carbon
```
Comment (ojcurtis) [inline .nvmrc]: Do we still need this file given we are setting the node version in `package.json`?

Comment (k0nserv) [reply .nvmrc]: The file is still here, just changed it to say `lts/erbium` instead

Comment (ojcurtis) [reply .nvmrc]: Yeah - but my question is do we still need it at all in the codebase?

Comment (k0nserv) [reply .nvmrc]: Yes to ensure we all get the right version by doing `nvm use`

**Pull request merged**